lib/apiservers/engine/backends/container_test.go

```diff
@@ -137,6 +143,14 @@ func NewMockContainerProxy() *MockContainerProxy {
 	}
 }
 
+func NewMockStorageProxy() *MockStorageProxy {
```

Comment:
Really need a constructor for this?

Reply:
Just following the same pattern as the NewMockContainerProxy().  Currently, it doesn't do anything, but I believe just following the same pattern will lead to easier to understand code once we fill out that mock proxy.

Reply:
consistency!!